### PR TITLE
Boxed-list info rows

### PR DESCRIPTION
**Pull request by PhilippKosarev**
The projects tab shows these changes best, but the other tabs are also changed.
# Before:
eam-old-projects

# After:
eam-new-projects

The rows are no longer selectable/activatable, but the text inside them is selectable, so it is much easier to copy it. The asset description wasn't fully visible if it was over a certain length, which is fixed with these changes. The logs are much easier to scroll through since it expands all of them, instead of having a scroll window inside a scroll window. Even though this version takes up a bunch more space than before, I think the new rows improve legibility by a lot.

I'm not experienced with rust so it may be worth checking out the changes there for any programming war crimes. (the create_details_row function has been replaced by create_info_row and create_widget_row)

@aknarts This PR is mainly for you to get a look at and see if it breaks anything for the rewrite, not expecting it to get it merged right now.

(Also not sure why git counted 158 file changes)

### CODE REVIEW

**Comment by aknarts** [comment]
I have just one question... why are there 158 modified files?:D

**Comment by PhilippKosarev** [comment]
I'm guessing I've somehow accidentally changed chmod permissions from 644 to 755 (on every single file) :P

Found this [blogpost](https://ajoz.github.io/2021/10/31/git-old-mode-new-mode/), going to try to undo it now using his method, hopefully it won't make it worse

**Review by aknarts — APPROVED** [review]

File: src/ui/widgets/logged_in/library/asset_detail.rs [hunk 1]
```diff
@@ -230,6 +230,8 @@ impl EpicAssetDetails {
             "close",
             clone!(@weak self as details => move |_, _| {
                 details.collapse();
+                println!("FRRR")
```
Comment (aknarts) [inline src/ui/widgets/logged_in/library/asset_detail.rs:233]: Just noticed this :D

Comment (PhilippKosarev) [reply src/ui/widgets/logged_in/library/asset_detail.rs:233]: Woopsie, bit of debug code slipped through :P

Will take care of that with the next PR

Comment (aknarts) [reply src/ui/widgets/logged_in/library/asset_detail.rs:233]: Do not worry about it, I removed it while merging it into my branch.

**Pull request merged**